### PR TITLE
new articles for az local-context

### PR DESCRIPTION
**Pull request by dbradish-microsoft**

### CODE REVIEW

File: docs-ref-conceptual/TOC.yml [hunk 1]
```diff
@@ -81,6 +81,8 @@
        aad, authentication, authenticate
   - name: Create virtual machines
     href: azure-cli-vm-tutorial.yml
+  - name: Use local context
```
Comment (arrownj) [inline docs-ref-conceptual/TOC.yml:84]: Let's use `param persist` instead, same with following name.

File: docs-ref-conceptual/azure-cli-configuration.md [hunk 2]
```diff
@@ -18,8 +18,9 @@ configuration file or with environment variables.
 Configuration values used by the CLI are evaluated in the following precedence, with items higher on the list taking priority.
 
 1. Command-line parameters
-2. Environment variables
-3. Values in the configuration file or set with `az configure`
+1. Values in the local working directory set by `az local-context`
```
Comment (arrownj) [inline docs-ref-conceptual/azure-cli-configuration.md:21]: `az config param-persist`

File: docs-ref-conceptual/azure-cli-local-context-tutorial.md [hunk 3]
```diff
@@ -0,0 +1,295 @@
+---
```
Comment (arrownj) [inline docs-ref-conceptual/azure-cli-local-context-tutorial.md:1]: Let's replace `local context` with `param persist`.

File: docs-ref-conceptual/azure-cli-local-context.md [hunk 4]
```diff
@@ -0,0 +1,180 @@
...
+Configuration values used by the CLI are evaluated in the following precedence, with items higher on the list taking priority.
+
+1. Command-line parameters
+1. Values in the local working directory set by `az local-context`
+1. Environment variables
+1. Values in the configuration file or set with `az configure`
+
+## Local context data file
+
+Local context data is kept in a file named `.local_context` which is stored in your working directory.  If you are using [Azure Cloud Shell](https://shell.azure.com) to execute Azure CLI commands, your working directory is in the storage account being used by the Azure CLI.  If you are using a [local install](/install-azure-cli) of the Azure CLI, your working directory is on your local machine.  In either case, the local context file is a hidden file and should not be manually updated.
```
Comment (arrownj) [inline docs-ref-conceptual/azure-cli-local-context.md:27]: The exact path is `./.azure/.local_context_<os_user_name>`

**Review by arrownj — COMMENTED** [review]
Hi @dbradish-microsoft , this doc is fantastic, I think it clearly explain how to run CLI commands with param persist. Just as my comments, as we have updated `local context` to `param persist`, so let's align the name in these docs. :)

**Comment by dbradish-microsoft** [comment]
Closing this PR as az local-context has been changed to az param-persist and these docs have been updated in another PR.

**Pull request closed without merging**